src/test/java/net/datafaker/providers/healthcare/MedicalProcedureTest.java

```diff
@@ -0,0 +1,17 @@
+package net.datafaker.providers.healthcare;
+
+import org.junit.jupiter.api.RepeatedTest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class MedicalProcedureTest extends HealthcareFakerTest {
+
+    private final MedicalProcedure medicalProcedure = getFaker().medicalProcedure();
+
+    @RepeatedTest(100)
+    void testProcedureCodes() {
+        // will use icd-10-pcs - https://regex101.com/library/nJ1wC4
```

Comment:
There is icd10-pcs, there's icd10-cm, and there's also icd10-am. Maybe it would be good to be explicit in the `icd10()` method which one we mean?

Reply:
To clarify it -- in the context here, that is `MedicalProcedure.icd10()`, we're testing for `icd-10-pcs` regex.

In future there would be `Disease.icd10()` which would test for `icd-10` regex (which is the same for all local variants).

Reply:
Ah yes, that makes sense now, sorry, I missed the class name, all good!